### PR TITLE
Add 3 icons: Up, Down, ErrorOutline

### PR DESCRIPTION
**Pull request by dskloetd**
# Motivation

We need these icons for the ckBTC design ([Figma](https://www.figma.com/file/x8dk5axQUf8Bi43uUsqCHc/NNS-Production-%26-Hand-over?type=design&node-id=482-6765&mode=design&t=8ebPRXhNeL9MYqFZ-0)).

# Changes

Added 3 icon files and added them to `src/lib/icons.ts`.

# Screenshots

image
image
image

### CODE REVIEW

File: src/lib/icons/IconDown.svelte [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  width={size}
+  viewBox="0 0 20 20"
+  fill="none"
+  xmlns="http://www.w3.org/2000/svg"
+>
+  <path
+    fill-rule="evenodd"
+    clip-rule="evenodd"
+    d="M10.7604 0L9.35472 0L9.35459 14.1319L4.09146 8.51553L2.99693 9.54121L9.51014 16.4916L10.0574 17.0756L10.6046 16.4916L17.1027 9.55789L16.0082 8.53218L10.7603 14.132L10.7604 0ZM17.9998 18V19.5L2 19.4998V17.9998L17.9998 18Z"
+    fill="currentColor"
```
Comment (peterpeterparker) [inline src/lib/icons/IconDown.svelte:18]: Does it work if your replace `fill=none` with `fill="currentColor"` and remove this tag?

i.e. space one attribute

File: src/lib/icons/IconErrorOutline.svelte [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width={size}
+  height={size}
+  viewBox="0 0 20 20"
+  fill="none"
+>
+  <path
+    stroke="currentColor"
```
Comment (peterpeterparker) [inline src/lib/icons/IconErrorOutline.svelte:16]: Does it work if you set the `stroke=currentColor` on the `svg` element?

i.e. space two attributes

File: src/lib/icons/IconUp.svelte [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  height={size}
+  width={size}
+  viewBox="0 0 20 20"
+  fill="none"
+  xmlns="http://www.w3.org/2000/svg"
+>
+  <path
+    fill-rule="evenodd"
+    clip-rule="evenodd"
+    fill="currentColor"
```
Comment (peterpeterparker) [inline src/lib/icons/IconUp.svelte:17]: ditto

**Review by peterpeterparker — APPROVED** [review]
LGTM, a minor question

File: src/lib/icons/IconDown.svelte (hunk 1 above)
Comment (dskloetd) [reply src/lib/icons/IconDown.svelte]: Yes, done.

File: src/lib/icons/IconErrorOutline.svelte (hunk 2 above)
Comment (dskloetd) [reply src/lib/icons/IconErrorOutline.svelte]: I've moved all the repeated attributes to the svg tag.

File: src/lib/icons/IconUp.svelte (hunk 3 above)
Comment (dskloetd) [reply src/lib/icons/IconUp.svelte]: done

**Pull request merged**